.github/workflows/run_tests.yml

```diff
@@ -54,32 +48,28 @@ jobs:
       - name: install non-py dependencies
         run: sudo apt-get update && sudo apt-get install -y libxml2-dev libxslt1-dev libpq-dev git gcc
 
-      - name: set up python${{ matrix.python-version }}
+      - name: Install poetry
+        run: pipx install poetry
```

Comment:
Should this be pinned to the same version as the Dockerfile (2.1.1)?  I don't have a strong opinion. It might be worth to leave unpinned as a canary.